src/test/java/org/folio/dew/batch/bursarfeesfines/service/BursarFilterEvaluatorTest.java

```diff
@@ -0,0 +1,305 @@
+      .build();
+
+    BursarExportFilterPass bursarExportFilterPass = new BursarExportFilterPass();
+
+    JsonNullable<BursarExportFilter> jsonNullableFilterPass = JsonNullable.of(
+      bursarExportFilterPass
+    );
+
+    assertTrue(
+      BursarFilterEvaluator.evaluate(
+        accountWithAncillaryData,
+        jsonNullableFilterPass
+      )
+    );
+
+    assertTrue(
+      BursarFilterEvaluator.evaluate(
+        accountWithAncillaryData,
+        JsonNullable.<BursarExportFilter>undefined()
+      )
+    );
+  }
+
+  @Test
+  void filterAccountByAge() {
+    Account account = new Account();
+    account.setDateCreated(new Date());
+
+    AccountWithAncillaryData accountWithAncillaryData = AccountWithAncillaryData
+      .builder()
+      .account(account)
+      .user(null)
+      .item(null)
+      .build();
+
+    BursarExportFilterAge filterAge = new BursarExportFilterAge();
+    filterAge.setNumDays(1);
+
+    assertFalse(
+      BursarFilterEvaluator.evaluate(accountWithAncillaryData, filterAge)
+    );
+
+    Calendar calendar = Calendar.getInstance();
+    calendar.add(Calendar.DAY_OF_MONTH, -2);
+
+    account.setDateCreated(calendar.getTime());
+    accountWithAncillaryData.setAccount(account);
+
+    assertTrue(
+      BursarFilterEvaluator.evaluate(accountWithAncillaryData, filterAge)
+    );
+  }
+
+  @Test
+  void filterAccountByAmount() {
+    BursarExportFilterAmount filterAmount = new BursarExportFilterAmount();
+
+    Account account = new Account();
+    account.setAmount(new BigDecimal(50));
+    AccountWithAncillaryData accountWithAncillaryData = AccountWithAncillaryData
+      .builder()
+      .account(account)
+      .user(null)
+      .item(null)
+      .build();
+
+    // test for accounts less than filter value
+    filterAmount.setAmount(6000);
+    filterAmount.setCondition(BursarExportFilterAmount.ConditionEnum.LESS_THAN);
+    assertTrue(
+      BursarFilterEvaluator.evaluate(accountWithAncillaryData, filterAmount)
+    );
+
+    // test for accounts less than or equal to filter value
+    filterAmount.setAmount(5000);
+    filterAmount.setCondition(
+      BursarExportFilterAmount.ConditionEnum.LESS_THAN_EQUAL
+    );
+    assertTrue(
+      BursarFilterEvaluator.evaluate(accountWithAncillaryData, filterAmount)
+    );
+
+    // test for accounts greater than filter value
+    filterAmount.setAmount(4000);
+    filterAmount.setCondition(
+      BursarExportFilterAmount.ConditionEnum.GREATER_THAN
+    );
+    assertTrue(
+      BursarFilterEvaluator.evaluate(accountWithAncillaryData, filterAmount)
+    );
+
+    // test for accounts less than filter value
+    filterAmount.setAmount(5000);
+    filterAmount.setCondition(
+      BursarExportFilterAmount.ConditionEnum.GREATER_THAN_EQUAL
+    );
+    assertTrue(
+      BursarFilterEvaluator.evaluate(accountWithAncillaryData, filterAmount)
+    );
+  }
+
+  @Test
+  void filterAccountByFeeType() {
+    UUID feeFineTypeUUID = UUID.randomUUID();
```

Comment:
Do not use randomized values in tests.  You can use something like `UUID.fromString("00000000-0000-4000-8000-000000000000")`